.devcontainer/Dockerfile

```diff
@@ -1,86 +1,89 @@
 FROM python:3.6
-LABEL maintainer “Shiho ASA<asashiho@mail.asa.yokohama>”
+LABEL maintainer "Shiho ASA<asashiho@mail.asa.yokohama>"
+
+# definitely the most annoying part is getting credentials
+COPY ./credentials /opt/aws
 
 # Install dependencies
 RUN apt-get update && apt-get install -y \
     apt-utils \
-    libblas-dev \
-    liblapack-dev\
-    libatlas-base-dev \
-    mecab \
-    mecab-naist-jdic \
-    mecab-ipadic-utf8 \
-    swig \
-    libmecab-dev \
-    gfortran \
+    # libblas-dev \
+    # liblapack-dev\
+    # libatlas-base-dev \
+    # mecab \
+    # mecab-naist-jdic \
+    # mecab-ipadic-utf8 \
+    # swig \
+    # libmecab-dev \
+    # gfortran \
 #    libav-tools \
     python3-setuptools \
     sudo \
     curl \
     vim \
-    mc \
-    build-essential \
-    libcairo2-dev \
-    libpango1.0-dev \
-    libjpeg-dev \
-    libgif-dev \
-    librsvg2-dev
-
+    build-essential
+    # mc \
+    # libcairo2-dev \
+    # libpango1.0-dev \
+    # libjpeg-dev \
+    # libgif-dev \
+    # librsvg2-dev
```

Comment:
Why are we removing those libs? If we remove them we will not be able to install canvas module locally.
That may be good idea ;). Just please keep mc package. I use it to verify content of tars and zips, debs without unpacking the compressed file. Very useful utility.